h2o-py/h2o/explanation/_explain.py

```diff
@@ -1235,7 +1235,8 @@ def ice_plot(
         max_levels=30,  # type: int
         figsize=(16, 9),  # type: Union[Tuple[float], List[float]]
         colormap="plasma",  # type: str
-        save_plot_path=None # type: Optional[str]
+        save_plot_path=None, # type: Optional[str]
+        binary_score_form=None # type: Literal["probability", "logodds"]
```

Comment:
wouldn't it be better to have just the following?
```suggestion
        binary_score_form="probability" # type: Literal["probability", "logodds"]
```

I think it would make it more clear about what is the default and you could then remove one condition in the function body (line 1301-1302). Otherwise the type declaration is incorrect (should be `Optional[Literal["probability", "logodds"]]`).

Reply:
Maybe I would go for Optional, because you don't want to use binary_score_form parameter for non binomial models

Reply:
1f6c8de

Reply:
[b127e35](https://github.com/h2oai/h2o-3/pull/6034/commits/b127e35e1145c6725d4c8527ee6d2e8397c35e03)

(changed to not optional after Erin's review)